### ISSUE TITLE
import properties generate Unexpected token error

### ISSUE THREAD

**Issue by ThreeDfish** [eslint/eslint#13511]
* **ESLint Version:** 7.5.0
* **Node Version:** 12.18.2
* **npm Version:** 6.14.5

**What parser (default, Babel-ESLint, etc.) are you using?**
default ecmaVersion:2019

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```json
{
    "parserOptions": {
		"sourceType": "module",
        "ecmaVersion": 2019,
        "ecmaFeatures": {
            "node": true
        }
	},
	"rules": {
		"no-inline-comments": "off"
	},
	"noInlineConfig": false,
	"env": {
		"es6": true,
		"node": true
	},
	"overrides": [
		{
			"files": ["*.mjs"],
			"parserOptions": {
				"sourceType": "module"
			},
			"rules": {
				"no-inline-comments": "off"
			}
		}
	]
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
import { dirname } from 'path';
import { fileURLToPath } from 'url';

const __dirname = dirname(fileURLToPath(import.meta.url));
```

```bash
eslint script.mjs
```

**What did you expect to happen?**
NO ERRORS

**What actually happened? Please include the actual, raw output from ESLint.**
Parsing error: Unexpected token import [4, 41]

**Are you willing to submit a pull request to fix this bug?**
If I knew how to fix it

**Related documents**
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import.meta

**Comment by mdjermanovic** [comment]
Hi @ThreeDfish, thanks for the issue!

`import.meta` is ECMAScript 2020 syntax, could you please try with `"ecmaVersion": 2020` in the configuration?

**Comment by ThreeDfish** [comment]
Thanks, I was able to get it to work with the 2020 settings.  I thought that was part of the 2019 spec.
